bolt-cli/src/contracts/bolt.rs

```diff
@@ -4,13 +4,25 @@ sol! {
     #[allow(missing_docs)]
     #[sol(rpc)]
     interface BoltValidators {
+        struct ValidatorInfo {
```

Comment:
you can add `#[derive(Debug, Serialize, Deserialize)]` and then encode it to JSON if necessary